packages/core/src/RenderingEngine/WSIViewport.ts

```diff
@@ -244,7 +254,7 @@ class WSIViewport extends Viewport {
       getDirection: () => metadata.direction,
       getDimensions: () => metadata.dimensions,
       getRange: () => [0, 255],
-      getScalarData: () => this.getScalarData(),
```

Comment:
Can you add a viewReference to the getter as an optional?  I know we don't pass it in yet, but we need to start thinking about it as the getScalarData for whole slide imaging won't be able to return the entire scalar data.